packages/block-library/src/video/edit.native.js

```diff
@@ -372,20 +372,14 @@ class VideoEdit extends Component {
 }
 
 export default compose( [
-	withDispatch( ( dispatch, { clientId }, { select } ) => {
-		return {
-			wasBlockJustInserted() {
-				const { clearLastBlockInserted } = dispatch( blockEditorStore );
-				const { wasBlockJustInserted } = select( blockEditorStore );
-
-				const result = wasBlockJustInserted( clientId );
+	withSelect( ( select, { clientId } ) => {
+		const { wasBlockJustInserted } = select( blockEditorStore );
 
-				if ( result ) {
-					clearLastBlockInserted();
-					return true;
-				}
-				return false;
-			},
+		return {
+			wasBlockJustInserted: wasBlockJustInserted(
+				clientId,
+				'inserter_menu'
+			),
 		};
```

Comment:
Not a blocker but I just noticed that the logic here could be converted to one line.`wasBlockJustInserted: select( blockEditorStore).wasBlockJustInserted( clientId, 'inserter_menu' )`

Reply:
Yep, I saw both patterns in different components so I guess it depends if you want to take multiple selectors from the same store. In this case since it's only one, I think we could reduce it to one line 👍 .

Reply:
I made this change in in this [commit](https://github.com/WordPress/gutenberg/pull/30238/commits/289e50616b1133db3d433cd2c6ea15b882e59cc0).